client/src/store/decks.js

```diff
@@ -0,0 +1,55 @@
+import { apiUrl } from '../config';
+
+// action types
+// get a collection of public decks
+export const LOAD_PUBLIC_DECKS = '/simple-decks/decks/LOAD_PUBLIC_DECKS';
+
+
+// action creators
+// get a collection of public decks
+export const loadPublicDecks = (decks) => ({
+  type: LOAD_PUBLIC_DECKS,
+  decks
+})
+
+// thunks
+// thunk for getting public decks
+export const loadPublicDecksThunk = () => async dispatch => {
+  const res = await fetch(`${apiUrl}/decks`);
+  const data = await res.json();
+  const decks = {};
+  data.forEach(deck => {
+    const { id, name, categoryId, userId, createdAt, updatedAt} = deck;
+    const privacy = deck.private;
+    const numCards = deck.Cards.length;
+    const category = deck.Category.label;
+    const creator = deck.User.username;
+    const maxScore = (deck.Scores.length) ? Math.max(deck.Scores) :null;
+    decks[id] = {
+      id,
+      name,
+      categoryId,
+      creatorId: userId,
+      privacy,
+      numCards,
+      category,
+      creator,
+      maxScore,
+      createdAt,
+      updatedAt,
+    }
+  })
```

Comment:
You should have your backend route structure this data before it's sent to the frontend, so that you have the `decks` data available to dispatch, similarly to how you did with the user login:

> const { user } = await response.json();
> dispatch(setUser(user));